spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/properties/RateLimitProperties.java

```diff
@@ -39,9 +39,9 @@
     private boolean behindProxy;
     private boolean enabled;
 
-    private Strategy strategy;
+    private Repository repository;
 
-    public enum Strategy {
+    public enum Repository {
         REDIS, CONSUL
```

Comment:
Add IN_MEMORY

Reply:
It's not really needed because I'm adding it directly on top of `InMemoryConfiguration` as a fallback.